external/mocap_optitrack/config/mocap_camera.yaml

```diff
@@ -0,0 +1,17 @@
+#
+# Definition of all trackable objects
+# Identifier corresponds to Trackable ID set in Tracking Tools
+#
+rigid_bodies:
+    '1':
+        pose: t265_pose_frame_pose
```

Comment:
Was this from trying things out?  No issues if we leave it here?

Reply:
Yup, its for terrain map testing but its its own file so it should be fine.